x/stakeibc/client/cli/tx_set_community_pool_rebate.go

```diff
@@ -0,0 +1,60 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/Stride-Labs/stride/v19/x/stakeibc/types"
+)
+
+func CmdSetCommunityPoolRebate() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set-rebate [chain-id] [rebate-percentage] [liquid-staked-amount]",
+		Short: "Registers or updates a community pool rebate",
+		Long: strings.TrimSpace(`Registers a community pool rebate by specifying the rebate percentage (as a decimal)
```

Comment:
same nit as a above - if I read this docstring, I would probably enter 20.0 instead 0.2 for 20%

Reply:
updated to `rebate_rate`